packages/validator/src/util/params.ts

```diff
@@ -121,14 +121,17 @@ function getSpecCriticalParams(localConfig: ChainConfig): Record<keyof ConfigWit
     MAX_PER_EPOCH_ACTIVATION_CHURN_LIMIT: denebForkRelevant,
     CHURN_LIMIT_QUOTIENT: true,
 
-    // Proposer boost
+    // Fork choice
     PROPOSER_SCORE_BOOST: false, // Ignored as it's changing https://github.com/ethereum/consensus-specs/pull/2895
 
     // Deposit contract
     DEPOSIT_CHAIN_ID: false, // Non-critical
     DEPOSIT_NETWORK_ID: false, // Non-critical
     DEPOSIT_CONTRACT_ADDRESS: true,
 
+    // Networking
+    MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: false,
```

Comment:
@g11tech Based on my understanding of this value it is not critical to verify it here as it is only relevant for the beacon node and as part of networking can't result in a consensus failure.

Maybe you could confirm this, and should we add a comment here?

Reply:
yes it doesn't affect consensus, adding with `false` is ok

Reply:
I added a comment in https://github.com/ChainSafe/lodestar/pull/6328/commits/e94355bd2189a3e035899fe0f4c4530a126eeabd, I'd assume this applies to all networking params. Also those are just relevant for the beacon node as far as I can see, so there is no reason the validator client needs to validate those